frontend/src/components/EntityList.vue

```diff
@@ -109,59 +61,56 @@ export default {
   },
   watch: {
     entitiesPerPage() {
-      this.getEntities({ resetPage: true });
+      this.getEntities({resetPage: true});
     },
+    schema() {
+      console.debug("Schema has changed?", this.schema)
+      if (this.schema === undefined || this.schema === null) {
+        console.debug("Oops, no valid schema")
+        return
+      }
+      this.getEntities();
+    }
   },
   methods: {
-    async selectSchema(schema) {
-      this.selectedSchema = schema;
-      this.filters = {};
-      const response = await api.getEntities({
-        schemaSlug: schema.slug,
-        limit: this.entitiesPerPage,
-        offset: this.offset,
-        meta: true,
-      });
-      const json = await response.json();
-      this.entities = json.entities;
-      this.totalEntities = json.total;
-      this.operators = json.meta.filter_fields.operators;
-      this.filterableFields = json.meta.filter_fields.fields;
```

Comment:
You forgot to move this part to `getEntities()` -- without this we don't have list of filterable fields and operators for them

Reply:
Oops, we definitely want to keep that. :+1: